hikari/api/rest.py

```diff
@@ -1766,6 +1766,8 @@ async def edit_my_user(
 
         Raises
         ------
+        hikari.errors.BadRequestError
```

Comment:
Might be worth it documenting the behaviour here with the sub-ratelimits which iirc exist and would lead to the ratelimited error being raised

Reply:
Those should also be documented for places like edit channel and execute webhook if they aren't already

Reply:
I dont think `edit_my_user` has any sub-ratelimits. iirc, its only `edit_channel` that has them

Reply:
edit_my_user is actually an outlier in the fact that it will return a 400 instead of a 429 if you change fields too quickly